Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 test_steps = [
 	'deb': { ->
 		stage('Ubuntu Xenial Debian Package') {
-			wrappedNode(label: 'docker-edge && x86_64', cleanWorkspace: true) {
+			wrappedNode(label: 'x86_64', cleanWorkspace: true) {
```

Comment:
would prefer to keep restriction of `ubuntu` at least. with just `x86_64` there are other types of nodes that it would run on that have caused issues in the past